iscript/src/iscript/macvpn.py

```diff
@@ -64,20 +65,133 @@ async def _sign_app(config, sign_config, app, entitlements_url, provisionprofile
     """
     # We mock the task for downloading entitlements and provisioning profiles
     entitlements_path = await download_entitlements_file(config, sign_config, {"payload": {"entitlements-url": entitlements_url}})
+
+    # TODO: Add provisionprofile_dir to scriptworker config
+    # https://github.com/mozilla-releng/scriptworker/blob/master/src/scriptworker/constants.py#L34
     provisioning_profile_path = None
-    if config.get("provisionprofile_dir"):
-        provisioning_profile_path = os.path.join(config["provisionprofile_dir"], provisionprofile_filename)
-        if not os.path.exists(provisioning_profile_path):
+
+    if provisionprofile_filename:
+        pp_dir = config.get("provisioning_profile_dir", None)
+        if not pp_dir:
+            pp_dir = Path(config["work_dir"]).parent / "provisionprofiles"
+            log.warning(f"No provisioning_profile_dir in settings, using default {pp_dir}")
+        provisioning_profile_path = Path(pp_dir) / provisionprofile_filename
+        if not provisioning_profile_path.is_file():
             log.error(f"Could not find provisionprofile file: {provisioning_profile_path}")
             raise IScriptError(f"Could not find provisionprofile file: {provisioning_profile_path}")
-        else:
-            log.info(f"Using provisionprofile {provisioning_profile_path}")
+        log.info(f"Using provisionprofile {provisioning_profile_path}")
 
     await sign_all_apps(config, sign_config, entitlements_path, [app], provisioning_profile_path)
 
     log.info(f"Done signing app {app.app_name}")
 
 
+async def _sign_util(sign_config, binary_path):
+    """Sign command for VPN util binaries.
```

Comment:
This function isn't actually specific to either the `utils` dir or binaries for that matter. Maybe it should just be called `_codesign`?